packages/web-pkg/src/composables/keyboardActions/useKeyboardActions.ts

```diff
@@ -0,0 +1,74 @@
+import { onBeforeUnmount, onMounted, ref } from 'vue'
+
+export enum Key {
+  C = 'c',
+  V = 'v',
+  A = 'a',
+  Space = ' ',
+  ArrowUp = 'ArrowUp',
+  ArrowDown = 'ArrowDown',
+  Esc = 'Escape'
+}
+
+export enum ModifierKey {
+  Ctrl = 'Control',
+  Shift = 'Shift'
+}
+
+export const useKeyboardActions = (keyBindOnElementId: string | null = null) => {
+  const actions = ref([])
+  const listener = (event: KeyboardEvent): void => {
+    event.preventDefault()
+    const { key, ctrlKey, metaKey, shiftKey } = event
+    const modifier = metaKey || ctrlKey ? ModifierKey.Ctrl : shiftKey ? ModifierKey.Shift : null
```

Comment:
Can you add parentheses here? I think this is rather hard to read

Reply:
Parentheses would get removed with linting, but I've just switched it classic if / else if, this should be more readable